src/components/Meta/index.js

```diff
@@ -1,7 +1,12 @@
 import Head from 'next/head'
+import { hotjar } from 'react-hotjar'
+import { useEffect } from 'react'
 
 const Meta = ({ title }) => {
   const applicationTitle = 'Hackney Repairs Hub'
+  useEffect(() => {
+    hotjar.initialize(783901, 6)
+  }, [])
```

Comment:
Surprised this didn't trigger a linter warning - normally it says something like:

```
Warning: React Hook useEffect has a missing dependency: 'hotjar.initialize'. Either include it or remove the dependency array.  react-hooks/exhaustive-deps
```

Maybe there's some difference in config between RHFE and BCFE 🤔

Reply:
If I had to guess it would be that BCFE has an extra linter extension in .eslintrc - "`next/core-web-vitals`" which RHFE does not have